src/shared/database/index.ts

```diff
@@ -484,55 +486,89 @@ function calculateAgreementScore(scores: AgreementScore[]): AgreementScore {
 }
 
 /**
- *  Updates a validator's 1 hour agreement score.
+ * Updates a validator's 1 hour agreement score.
  *
- * @param master_key - Signing key of the the validator to be updated.
+ * @param validator_keys - Signing keys of the the validator to be updated.
  * @param agreement - An agreement object.
  * @returns A promise that resolves to void once the agreement has been stored.
  */
 export async function update1HourValidatorAgreement(
-  master_key: string,
+  validator_keys: ValidatorKeys,
   agreement: AgreementScore,
 ): Promise<void> {
-  await query('validators')
-    .where({ master_key })
-    .update({ agreement_1hour: agreement })
-    .catch((err) => log.error('Error Updating 1 Hour Validator Agreement', err))
+  const { master_key, signing_key } = validator_keys
+  if (master_key) {
+    await query('validators')
+      .where({ master_key })
+      .update({ agreement_1hour: agreement })
+      .catch((err) =>
+        log.error('Error Updating 1 Hour Validator Agreement', err),
```

Comment:
Should these errors be slightly different so it's easy to tell whether it checked the `master_key` or `signing_key`? (Ex. `Error Updating 1 Hour Validator Agreement for master_key ${master_key}`